logging/config/cluster-logging-external.adoc

```diff
@@ -1,17 +1,17 @@
 :context: cluster-logging-external
 [id="cluster-logging-external"]
-= Sending logs to external devices using Fluentd Forward plug-ins
+= Sending logs to external Fluentd and Rsyslog devices using plug-ins
```

Comment:
I'm not sure how to word this.  @jcantrill @aconway @sichvoge This document explains how to configure cluster logging to send logs from the cluster logging Fluentd to another logging product.

The *out_forward* plug-in is used to send logs securely to another Fluentd.  This was formerly known as *secure_forward* in 3.x.  The protocol is based on msgpack and is specific to Fluentd.

The *fluent-plugin-remote-syslog* plugin is used to send logs to another logging product using the syslog (RFC3164) protocol.  Many logging products such as Fluentd, Rsyslog, and others support this protocol.  The advantage of the syslog protocol is that it is widely used - can be used to send logs to many different products.  The disadvantages are:
* not secure - no support for TLS
* no metadata - no way to provide Kubernetes metadata, systemd data, or other metadata

(We are looking for a different implementation that supports TLS, and supports RFC5424 which would allow us to provide all of the metadata gathered - but in any case, it really depends on if theremote syslog logging product supports TLS and RFC5424).

Reply:
@richm Maybe it should be "Sending logs externally using syslog RFC3164"  Does that properly capture the facts:
* unsecure
* no meta
* anyone who understands RFC3164

Reply:
it is possible to use TLS + RFC3164 -  I believe rsyslog can do that, among others - but the *fluent-plugin-remote-syslog* cannot do TLS.